practice_app/services/getLocationInfo.spec.js

```diff
@@ -1,4 +1,174 @@
 const { processLocationInfo } = require('./getLocationInfo')
+const { getLanguage } = require('./getLocationInfo')
+const { getCurrency } = require('./getLocationInfo')
+const { getCountry } = require('./getLocationInfo')
```

Comment:
Import all functions in the same line with `const { processLocationInfo, getLanguage, getCountry, getCurrency } = require('./getLocationInfo')`

Reply:
I have fixed this